src/widgets/copilot/components/Messages/LocationModal.tsx

```
@@ -0,0 +1,443 @@
+import clsx from "clsx";
+
+import LocationRefresh from "src/assets/SvgIcons/LocationRefresh";
+import LocationMarker from "src/assets/SvgIcons/LocationMarker";
+import Spinner from "src/assets/SvgIcons/Spinner";
+
+import Button from "src/components/shared/Buttons/Button";
+import { ReplyButton } from "./IncomingMsg";
+import IconButton from "src/components/shared/Buttons/IconButton";
+
+import L from "leaflet";
+import { APIProvider, Map, useMap } from "@vis.gl/react-google-maps";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)